signup-client.html

```diff
@@ -44,21 +46,17 @@ <h1>Client Sign Up</h1>
                 <input type="password" placeholder="repeat password" id="password-repeat" name="password-repeat"
                     required><br>
 
-            <!--sign up button-->
-            <button type="submit" onsubmit="saveCustomer()">Submit</button>
+                <!--sign up button-->
```

Comment:
is not up to date with the one on main . copy the one from main for all this file